build-system/global-configs/experiments-config.json

```diff
@@ -1,5 +1,19 @@
 {
-  "experimentA": {},
-  "experimentB": {},
+  "experimentA": {
+    "name": "IntersectionObserverPolyfill",
+    "environment": "AMP",
+    "command": "gulp dist --defineExperimentConstant=INTERSECTION_OBSERVER_POLYFILL",
+    "issue": "https://github.com/ampproject/amphtml/issues/27807",
+    "expirationDateUTC": "2020-05-31",
+    "defineExperimentConstant": "INTERSECTION_OBSERVER_POLYFILL"
+  },
+  "experimentB": {
+    "name": "IntersectionObserverPolyfill",
+    "environment": "INABOX",
```

Comment:
Should we declare the experiment after we have hookup 3p host updater for inabox?

Reply:
Do you mean you'd prefer if I declared this experiment later?

Reply:
My understanding is that without setting up the message channel with the host, the new InObPolyfill won't work within inabox. And we will be using the old ones anyway. If that's the case I'd prefer us declare the experiment afterwards. (experiment binary need to be built and with every master build, require some extra resources on travis)

Correct me if the InObPolyfill is used in inabox.

Reply:
The inabox will stay on the old polyfill for a little while. But not very long. The new implementation for inabox is also on the way. I removed the experiment for now, however. I will send it with an appropriate pull request.

Reply:
Thank you for the clarification.